### DISCUSSION TITLE
Is it possible to add "Sloc" user customization?

### DISCUSSION THREAD

**Discussion by IAm0xAxe** [solodit/feedback#37] [New Feature Requests]
I noticed that there are a lot of races in C4 where "Sloc" has not been added, is it possible to add a "Sloc" user customizable feature? I would like to make a small contribution to this.

![image](https://github.com/solodit/feedback/assets/33568293/f52ad455-0585-40b9-a48f-81e79e92a68e)

**Comment by wang1252** [comment]
@IAm0xAxe 

Could you please make the CSV for the contests which has no sloc(maybe C4 contests)?
Then we will update it.

**Reply by IAm0xAxe** [reply]
@wang1252 

I had done something similar before:  https://github.com/orgs/solodit/discussions/3 .
But the image link is not working. Currently I need some time to recreate it. The process might be a bit slow ~ 
sorry about that.
